src/main/java/Task.java

```diff
@@ -0,0 +1,29 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public static Task createTask(String[] commandArgs) throws IllegalArgumentException {return null;};
```

Comment:
You shouldn't have the code for this constructor all in one line. You should have only one statement per line :)